### ISSUE TITLE
Add nvidia driver to initramfs to make plymouth work

### ISSUE THREAD

**Issue by freundTech** [ublue-os/hwe#249]
I recently installed `ublue-os/silverblue-nvidia:40` on a computer with an Nvidia RTX 3050.
At first everything seemed to work, however, after running `ujust configure-nvidia` to enable the proprietary drivers the plymouth boot animation was broken.

This can be fixed by adding the nvidia driver to the initramfs using dracut.

rpm-ostree usually uses an initramfs provided by the remote, but allows locally overwriting it.

To fix this issue ublue-os should either include the nvidia driver into the initramfs upstream or configure ujust to add the modules and enable local initramfs rebuilding.

The driver can be added by adding `/etc/dracut.conf.d/nvidia.conf` with content
```
add_drivers+=" nvidia nvidia_modeset nvidia_uvm nvidia_drm "
```
When this is done upstream before the initramfs is generated nothing else has to be done.

When letting ujust do this it also need to run `rpm-ostree initramfs --enable` to enable local rebuilding of the initramfs.

**Comment by m2Giles** [comment]
We're adding this in now. Especially since Nvidia 555 needs it for Wayland.

We will also generate the initramfs in the build pipeline.

**Comment by m2Giles** [comment]
This has now been merged in #258
